pages/pricing.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react'
+import {
+	AiOutlineCheckCircle
+} from 'react-icons/ai'
+import Navbar from '../views/Navbar';
+import Footer from '../components/Footer';
+import Link from 'next/link';
+
+const pricingPlan = [
+	{
+		pricingName: 'Free',
+		duration: '.',
+		pricing: 'for open source projects',
+		features: ['Access of all features', 'Unlimited team size', 'For both Github and Bitbucket'],
+		buttonText: 'Add a Repo',
+		link: 'https://github.com/'
+
+	},
+	{
+		pricingName: 'Standard',
+		duration: 'per user/month',
+		pricing: '$100',
+		features: ['Access of all features', 'personal team support for setting up'],
+		buttonText: 'Start your 30/day trial',
+		link: 'https://chrome.google.com/webstore/detail/vibinex/jafgelpkkkopeaefadkdjcmnicgpcncc' // temp. adding chrome extension link, need to remove it with payment link
+
+	},
+	{
+		pricingName: 'Enterprise',
+		duration: '.',
+		pricing: 'custom pricing',
+		features: ['Good for large teams', 'personal team support for setting up', 'upto 2 custom features'],
+		buttonText: 'Contact Us',
+		link: 'https://api.whatsapp.com/send/?phone=918511557566&text&type=phone_number&app_absent=0'
+
+	},
+
+]
+
+
+const Pricing = () => {
+	const [priceDuration, setPriceDuration] = useState(false); // false for monthly and true for yearly
+	const [pricePlans, setPricePlans] = useState(pricingPlan);
+	const chromeExtensionLink = "https://chrome.google.com/webstore/detail/vibinex/jafgelpkkkopeaefadkdjcmnicgpcncc";
```

Comment:
Hmm... I think we are hard-coding this value too many times. We should make it a globally accessible value; for example, we can store it in the Context or, maybe, as an environment variable.

Reply:
will export it, but since in this pr, i think we need to use calendly link, which is good. so i'm removing it. 
also msg me what you discuessed in standup. related to adding whatsapp link or calendly link.